docs/projects/automate-project-git-source.ipynb

```diff
@@ -141,11 +141,16 @@
    "source": [
```

Comment:
<p> </p><blockquote>You must include <code>init_git=True</code> when creating a project so that you can then later <a href="https://app.reviewnb.com/mlrun/mlrun/pull/4733/#creating-and-pushing-changes-to-your-git-repo-or-archive-file" target="_blank">create the git remote</a>. </blockquote><p>That is no longer true, it was fixed in 1.6.0 (you can create a remote even if you didn't use init_git</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4733/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>